include/swift/Basic/Features.def

```diff
@@ -410,7 +410,7 @@ EXPERIMENTAL_FEATURE(SafeInterop, true)
 EXPERIMENTAL_FEATURE(AssumeResilientCxxTypes, true)
 
 // Isolated deinit
-SUPPRESSIBLE_EXPERIMENTAL_FEATURE(IsolatedDeinit, true)
+SUPPRESSIBLE_LANGUAGE_FEATURE(IsolatedDeinit, 371, "isolated deinit")
```

Comment:
Please keep this feature experimental on 6.1. We should only cherry pick the runtime ptrauth fix so that in a later release, the feature can back deploy to the 6.1 runtime.

Reply:
ok let me do that

Reply:
Hope I got that right, let's see https://github.com/swiftlang/swift/pull/78107/commits/8e61e8a39dfb04a1ca88c44c40bf7aca6e0a3523